airflow/cli/commands/provider_command.py

```diff
@@ -63,4 +69,15 @@ def provider_get(args):
 
 def providers_list(args):
     """Lists all providers at the command line"""
-    print(_tabulate_providers(ProvidersManager().providers.values(), args.output))
+    console = Console()
+    table = SimpleTable(title="Installed providers")
+    table.add_column("Name")
+    table.add_column("Description")
+    table.add_column("Version")
+    for _, provider in ProvidersManager().providers.values():
+        table.add_row(
+            provider['package-name'],
+            _remove_rst_syntax(provider['description']),
```

Comment:
```suggestion
           provider['name'],
```
Do we need to display the full description? Maybe the name is enough?

Reply:
The description has a url which can be helpful. However, I'm leaning towards showing only provider + version. On the other hand that information is also available via `airflow info`